enterprise-suite/console-api/prometheus.yml

```diff
@@ -84,9 +84,10 @@ scrape_configs:
         regex: (.*)-[0-9]{13}-(exec|driver)-[0-9]+;
         replacement: ${1}-${2}
         target_label: es_workload
-    # flink workload — of the form 'taxi-ride-fare-seglo-processor-dde77c7d-jm' or 'taxi-ride-fare-seglo-processor-dde77c7d-tm'
-      - source_labels: [kubernetes_pod_name, es_workload]
-        regex: (.*)-[a-z,0-9]{8}-(jm|tm);
+      # flink workload — of the form 'taxi-ride-fare-seglo-processor-dde77c7d-jm' or 'taxi-ride-fare-seglo-processor-dde77c7d-tm'
+      - source_labels: [kubernetes_pod_name]
```

Comment:
Can you add back the `, es_workload`? It probably doesn't matter much here, but it maintains consistency of the ordering precedence these matching are using (e.g. if it matches an earlier rule, it won't then match this rule).

Reply:
It looked like rules match in a cascade.  One rule can match, then a subsequent one, etc.  It was difficult to come up with a rule that output the right value to `es_workload` for flink because it can match on the general pod rule as well.

We could add `es_workload` back, but then the regex would just become the following, to skip matching on `es_workload` values that were produced from before.

```
;(.*)-[a-z,0-9]{8}-(jm|tm)-[456789bcdf]{4,10}-[bcdfghjklmnpqrstvwxz2456789]{5}
```